docs/grafana.rst

```diff
@@ -0,0 +1,320 @@
+Real time graphs
+====================
+
+Grafana
+-----------
+
+`Grafana <https://grafana.com/>`_ is the open source analytics and monitoring solution and may be used to view various BEAM metrics in real time during BEAM execution.
+
+Grafana is a tool to display data from data source. In our case the data source is `InfluxDB <https://www.influxdata.com/products/influxdb-overview/>`_ .
+
+
+Docker installation and configuration
+--------------------------------------
+
+There is a docker image with configured Grafana and InfluxDB to work with BEAM thus one need to install docker in order to use Grafana.
+
+Ubuntu Linux users
+^^^^^^^^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* 64-bit version of one of these Ubuntu versions: Disco 19.04, Cosmic 18.10, Bionic 18.04 (LTS), Xenial 16.04 (LTS)
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Ubuntu Linux users <https://docs.docker.com/install/linux/docker-ce/ubuntu/>`_ and
+a `Post-installation guide for Linux <https://docs.docker.com/install/linux/linux-postinstall/>`_ with optional procedures
+for configuring Linux hosts to work better with Docker for non-root users.
+
+Otherwise use those commands::
```

Comment:
Should this be the first thing mentioned since it seems simplest? And it would be these, not those.

Reply:
I would like people use official documentation.
Commands might became deprecated and we might not catch moment to fix them. 
@JustinPihony, maybe it is a good idea to completely remove these command and direct links for Win/Mac users?

Reply:
I'm ambivalent - your explanation is good to me - as to the commands - your call

Reply:
ok, if we both unsure, then let's save some energy and keep this part as it is